packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -548,7 +547,7 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
       const bip44Params = this.getBIP44Params({ accountNumber })
       const txToSign = {
         addressNList: toAddressNList(bip44Params),
-        value: convertNumberToHex(value),
+        value: numberToHex(value),
```

Comment:
ty, that was my bad copying this one from web (which itself could do with the same love using `numberToHex`)